### ISSUE TITLE
Android support?

### ISSUE THREAD

**Issue by mi12092** [alfred-gw/torirc#2]
Is it possible for people using android somehow connect to my torirc server on my pc trough Orbot and some other irc client that supports orbot proxy?

**Comment by ortegaalfredo** [comment]
No, torirc does not uses the IRC protocol, so IRC clients cannot connect to it. Also, torirc uses stem and orbot does not support that lib, AFAIK.

**Comment by RoPe93** [comment]
ok. thanks.
